src/components/form-field/text-input/text-input.tsx

```diff
@@ -60,10 +60,10 @@ export const TextInput = ({
         <div className={classNames("relative w-full", formFieldGroupStyles)}>
             {LeftIcon ? (
                 <div
-                    className="pointer-events-none absolute inset-y-0 left-0 flex items-center pl-3"
+                    className="pointer-events-none absolute inset-y-0 left-0 flex items-center pl-3 z-20"
```

Comment:
Could we also use `z-10` instead of `z-20` here? ☺️ And I just have added a little story for inputs with left icons :-)

Reply:
I tried first with `z-10`, but then the icon disappears when the input is focussed, therefore the `z-20` 😄